elasticsearch-singledocument/src/test/resources/log4j.xml

```diff
@@ -14,10 +14,6 @@
         </layout>
     </appender>
 
-    <logger name="org.vertexium.query.Query.QUERY" additivity="false">
```

Comment:
This logger was actually the source of my log4j issue yesterday. If you look 3 loggers down in this file, you'll find that it has a duplicate. I just removed the extra one to make log4j happy.